src/common/stores/store-names.ts

```diff
@@ -15,4 +15,5 @@ export enum StoreNames {
     UserConfigurationStore,
     PathSnippetStore,
     UnifiedScanResultStore,
+    DeviceStore,
```

Comment:
it might be better to group electron store names

Reply:
that sounds like a good idea and I like to implement it but I'll leave it for a different PR, the main reason is the base store implementation constructor receives `storeName: StoreNames`, i.e. we are using this enum as a proper type on some places and that will need a little more than spliting the file: 
```typescript
export abstract class BaseStoreImpl<TState> extends Store implements BaseStore<TState> {
    ...
    constructor(storeName: StoreNames) {
       ...
    }
```